docs/advanced/aliases.md

```diff
@@ -1,121 +1,132 @@
+## EightshiftBlocksSwiper
 
-#### Usage
-```js
-import { dynamicImport } from 'EighshiftBlocksDynamicImport';
+Alias providing [Swiper](https://www.npmjs.com/package/swiper).
 
-dynamicImport(require.context('./../../custom', true, /assets\/index.js$/));
+```js
+import 'EightshiftBlocksSwiper';
 ```
 
-## JavaScript Ucfirst Helper
+## EightshiftBlocksSwiperIE
 
-Converts the first letter of the string to uppercase.
+Alias providing [Swiper](https://www.npmjs.com/package/swiper) but for IE11 version.
 
-#### Usage
 ```js
-import { ucfirst } from 'EighshiftBlocksUcfirst';
-
-ucfirst('custom string');
+import 'EightshiftBlocksSwiperIE';
 ```
 
-## JavaScript Utilities path
+## EightshiftBlocksBabelPolyfill
 
-We have also created a bunch of JavaScript helpers located in [helpers folder](https://github.com/infinum/eightshift-frontend-libs/tree/develop/scripts/helpers).
-
-This alias provides only path to helpers folder.
+Alias providing [Babel Polyfill](https://babeljs.io/docs/en/babel-polyfill).
 
-#### Usage
 ```js
-import { media } from 'EighshiftBlocksUtilityHelpersPath/media';
+import 'EightshiftBlocksBabelPolyfill';
 ```
 
-## JavaScript WhatwgFetch
+## EightshiftBlocksAutoprefixer
 
-Alias providing [Whatwg-fetch](https://www.npmjs.com/package/whatwg-fetch). Check documentation for more details.
+Alias providing [Autoprefixer](https://github.com/postcss/autoprefixer).
 
-#### Usage
 ```js
-import 'EighshiftBlocksWhatwgFetch';
+import 'EightshiftBlocksAutoprefixer';
 ```
 
-## JavaScript Swiper
+## EightshiftBlocksNormalize
 
-Alias providing [Swiper](https://www.npmjs.com/package/swiper). Check documentation for more details.
-
-#### Usage
+Alias providing [Normalize](https://www.npmjs.com/package/normalize-scss). Check documentation for more details.
 
-JavaScript
-```js
-import 'EighshiftBlocksSwiper';
+```scss
+@import 'EightshiftBlocksNormalize';
+@include normalize(); // stylelint-disable-line at-rule-empty-line-before
 ```
 
-Styles:
+## EightshiftBlocksMediaBlender
+
+Alias providing [Media Blender](https://github.com/infinum/media-blender). Check documentation for more details.
+
 ```scss
-@import 'EighshiftBlocksSwiperStyle';
+@import 'EightshiftBlocksMediaBlender';
 ```
 
-## JavaScript Babel Polyfill
+## EightshiftBlocksSwiperStyle
+
+Alias providing [Swiper](https://www.npmjs.com/package/swiper).
+
+```scss
+@import 'EightshiftBlocksSwiperStyle';
+```
 
-Alias providing [Babel Polyfill](https://babeljs.io/docs/en/babel-polyfill). Check documentation for more details.
+## EightshiftBlocksDynamicImport
 
-#### Usage
+To get (require) all the files using `require.context`  method, we created a helper located [here](https://github.com/infinum/eightshift-frontend-libs/blob/develop/scripts/dynamic-import.js).
 
+It finds all files recursively in the folder using regex. This example will require all `assets/index.js` files inside the `custom` folder, so there is no need to add the files to the build manually.
```

Comment:
```suggestion
It will find all files recursively in the folder using a regex. The following example will require all `assets/index.js` files inside the `custom` folder, so there is no need to add the files to the build manually.
```